### DISCUSSION TITLE
csv import error，No detailed error information

### DISCUSSION THREAD

**Discussion by hopepanwei** [apache/doris#12656] [A - General / Q&A]
error msg : 
```
Reason: Unable to display. src line [Unable to display]; 
```
[test.csv](https://github.com/apache/doris/files/9580084/test.csv)

Only this row of data reported an error, and others were successfully imported

ddl：
create table ORG_CHA_INFO_RELA
(
    ID           VARCHAR(36)   not null,
    ISVALID      TINYINT	not null default "1",
    CREATETIME   DateTime not null,
    UPDATETIME   DateTime not null,
    CCXEID       DateTime not null,
    ORG_UNI_CODE BIGINT	not null,
    SEQ_NUM      INT	not null,
    CHA_DATE     DATE,
    CHA_PROJ     VARCHAR(1500),
    TYC_ORG_CODE BIGINT,
    TYC_ID       BIGINT,
    SHOW_STA     tinyint,
    CHA_AGO      STRING,
    CHA_AFTER    STRING
)
ENGINE=OLAP
DUPLICATE KEY(`ID`,`ISVALID`,`CREATETIME`,`UPDATETIME`,`CCXEID`,`ORG_UNI_CODE`,
`SEQ_NUM`,`CHA_DATE`,`CHA_PROJ`,`TYC_ORG_CODE`,`TYC_ID`,`SHOW_STA`)
DISTRIBUTED BY HASH (`ORG_UNI_CODE`) BUCKETS 10
PROPERTIES(
    "replication_num" = "1"
);

**Comment by JNSimba** [comment]
This error is usually because there are non-UTF8 characters in your file. If not, it may be a problem with the file. Can you try copying the content of the file into a new file?

**Comment by liangrui1988** [comment]
There are two problems with this
1.Vim text copy Example data, = fileencoding = cp936

2:The exception log is not clear, and we have no idea what the exception is...
Reason: Unable to display. src line [Unable to display];
Reason: Unable to display. src line [Unable to display];
Reason: Unable to display. src line [Unable to display];
Reason: Unable to display. src line [Unable to display];
Reason: Unable to display. src line [Unable to display];
Reason: Unable to display. src line [Unable to display];
Reason: Unable to display. src line [Unable to display];
